app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -190,118 +191,159 @@ export const ContentForm: React.FC<IContentFormProps> = ({ contentType = Content
             {(props) => (
               <Col>
                 <FormikHidden name="uid" />
-                <Row alignItems="flex-start">
-                  <Col flex="1.5 1 0%">
-                    <FormikText
-                      name="headline"
-                      required
-                      label="Headline"
-                      value={props.values.headline}
-                    />
-                    <Row>
-                      <FormikSelect
-                        name="dataSourceId"
-                        label="Source"
-                        width={FieldSize.Big}
-                        value={
-                          dataSourceOptions.find((mt) => mt.value === props.values.dataSourceId) ??
-                          ''
-                        }
-                        onChange={(newValue: any) => {
-                          const dataSource = dataSources.find((ds) => ds.id === newValue.value);
-                          props.setFieldValue('dataSourceId', newValue.value);
-                          props.setFieldValue('source', dataSource?.code ?? '');
-                          if (!!dataSource) {
-                            props.setFieldValue('mediaTypeId', dataSource.mediaTypeId);
-                            props.setFieldValue('licenseId', dataSource.licenseId);
-                          }
-                        }}
-                        options={dataSourceOptions}
-                        required={!props.values.source}
-                        isDisabled={!!props.values.otherSource}
-                      />
-                      <FormikHidden name="source" />
-                      <FormikText
-                        name="otherSource"
-                        label="Other Source"
-                        onChange={(e) => {
-                          const value = e.currentTarget.value;
-                          props.setFieldValue('otherSource', value);
-                          props.setFieldValue('source', value);
-                          if (!!value) {
-                            props.setFieldValue('dataSourceId', undefined);
-                          }
-                        }}
-                        required={!!props.values.otherSource}
-                      />
+                <Row alignItems="flex-start" className="content-details">
+                  <Show visible={size === 0}>
```

Comment:
Just a simple way to get more vertical space back on the screen when editing clips.